sdks/web-ui-sdk/src/main.ts

```diff
@@ -23,54 +46,35 @@ export const flows: BallerineSDKFlows = {
       // Always init with no state. This ensures using init to reset the flow returns to the first step with no data.
       void resetAppState();
 
-      configuration.set(defaultConfiguration);
-
-      const { translations: _translations, ...configWithoutTranslations } = config;
-
-      // Extract config from query params
-      const { clientId, flowName, ...endUserInfoFromQueryParams } = getConfigFromQueryParams();
-
-      // Merge the two config objects
-      const mergedConfig: FlowsInitOptions = {
-        ...configWithoutTranslations,
-        endUserInfo: {
-          ...configWithoutTranslations.endUserInfo,
-          ...endUserInfoFromQueryParams,
-        },
-      };
-
-      const configPromise = appInit(mergedConfig);
-      const translationsPromise = mergeTranslationsOverrides(_translations);
-
-      Promise.all([configPromise, translationsPromise])
+      initConfigAndTranslations(config)
```

Comment:
The `init` method is already the abstraction that encapsulates the config and translation initialization. It would make more sense if there was room to divide the init config logic from the init translations. Let's revert this change. 🙂